Lib/test/test_typing.py

```diff
@@ -4091,6 +4091,20 @@ class C(Generic[T]): pass
         with self.assertRaises(TypeError):
             C[()]
 
+    def test_generic_subclass_checks(self):
+        for typ in [list[int], List[int], tuple[int, str], Tuple[int, str],
+                    dict[str, int], Dict[str, int], set[str], typing.Set[str]]:
+            with self.subTest(typ=typ):
+                self.assertRaises(TypeError, issubclass, typ, object)
+                self.assertRaises(TypeError, issubclass, typ, type)
+                self.assertRaises(TypeError, issubclass, typ, typ)
+
+                # isinstance is fine:
+                self.assertTrue(isinstance(typ, object))
+                self.assertTrue(isinstance(typ, object))
```

Comment:
Duplicated lines. In any case, I do not think it is a right place for these tests. I am sure it is already tested anywhere.

Reply:
Fixed.